syntheseus/cli/search.py

```diff
@@ -304,10 +305,49 @@ def build_node_evaluator(key: str) -> None:
         with open(results_dir_current_run / "stats.json", "wt") as f_combined_stats:
             f_combined_stats.write(json.dumps(combined_stats, indent=2))
 
+    return results_dir_current_run
```

Comment:
As the exact directory is constructed on the spot (e.g. contains the timestamp), returning it is useful if one wants to call `search.py` from another script, e.g.
```python
results_dir = search.main(argv=None)

# Consume and analyze the results in `results_dir`.
with open(results_dir / "stats.json", "rt") as f_combined_stats:
    print(json.load(f_combined_stats))
    ...
```